### PR TITLE
Improved readability and detatched some commands from their output

### PR DESCRIPTION
**Pull request by mbianchidev**
### Description

I improved the readability of some commands separating their outputs, facilitating user copy-paste flow when trying out when trying the autoscaling setup, clarified the k6s installation is required before running the command and its javascript test.

### Testing instructions

Preview link: 

### Checklist 

- [ ] Review label added 
- [ ] [Conditional version tags](https://docs.konghq.com/contributing/conditional-rendering/#conditionally-render-content-by-version) added, if applicable.

### CODE REVIEW

File: app/_src/gateway-operator/guides/autoscaling-kong.md [hunk 1]
```diff
@@ -99,10 +106,9 @@ You can test if the autoscaling works by using a load testing tool (e.g. k6s) to
     export PROXY_IP=$(kubectl get dataplanes.gateway-operator.konghq.com -o jsonpath='{.status.addresses[0].value}' horizontal-autoscaling)
     ```
 
-1. Create a [`k6s`](https://k6.io/) configuration file.
+1. Install [`k6s`](https://k6.io/) then create a configuration file containing the following code:
```
Comment (lena-larionova) [inline app/_src/gateway-operator/guides/autoscaling-kong.md:109]:
```suggestion
1. Install [`k6s`](https://k6.io/), then create a configuration file containing the following code:
```

File: app/_src/gateway-operator/guides/autoscaling-kong.md [hunk 2]
```diff
@@ -123,13 +129,17 @@ You can test if the autoscaling works by using a load testing tool (e.g. k6s) to
...
    ```
 
 1. Observe the scaling events in the cluster while the test is running.
 
     ```bash
-    $ kubectl get events --field-selector involvedObject.name=horizontal-autoscaling --field-selector involvedObject.kind=HorizontalPodAutoscaler
+    kubectl get events --field-selector involvedObject.name=horizontal-autoscaling --field-selector involvedObject.kind=HorizontalPodAutoscaler
+    ```
+
+    The output will show the scaling events:
```
Comment (lena-larionova) [inline app/_src/gateway-operator/guides/autoscaling-kong.md:141]:
```suggestion
    The output shows the scaling events:
```

File: app/_src/gateway-operator/guides/autoscaling-kong.md [hunk 3]
```diff
@@ -75,7 +76,8 @@ spec:
             limits:
               memory: "1024Mi"
               cpu: "1000m"
-          # Konnect related environment variables, volumes etc...
+          # Add here any Konnect related environment variables, volumes etc...
```
Comment (lena-larionova) [inline app/_src/gateway-operator/guides/autoscaling-kong.md:79]:
```suggestion
          # Add any Konnect-related configuration here: environment variables, volumes, and so on.
```

**Review by lena-larionova — COMMENTED** [review]
A couple of minor formatting suggestions.

**Comment by mbianchidev** [comment]
> A couple of minor formatting & phrasing suggestions.

Done! Thanks for reviewing :)

**Review by lena-larionova — APPROVED** [review]
LGTM!

**Pull request merged**